### PR TITLE
I41 admin dashboard responsiveness

### PR DESCRIPTION
**Pull request by kirkkwang**
[Collapse sidebar at 768px](https://github.com/scientist-softserv/utk-hyku/commit/fbed225d1dd1d563117464d8077c9e17272c0bec) 

This commit will make the admin dashboard's sidebar collapse at anything
less than 768px. This is the same breakpoint used by Bootstrap to switch
to a smaller layout.

---

[Add title attribute](https://github.com/scientist-softserv/utk-hyku/commit/0ab2154d3f2768a1398de6a9b852e84f55de3fcb) 

This commit will add title attribute to sidebar links for hover
information display.  Some translations were missing in the yml so I
added those as well but only for English.  At some point we might want
to add support for other languages.  Bulkrax and AllinsonFlex had some
of the partials in the sidebar so those were also modified and the
revisions for those gems were updated.

Ref:
  - https://github.com/samvera-labs/allinson_flex/pull/120
  - https://github.com/samvera-labs/bulkrax/pull/838
  - https://github.com/scientist-softserv/utk-hyku/issues/41

---

[⬆️ Update Bulkrax and AllinsonFlex](https://github.com/scientist-softserv/utk-hyku/commit/d2b2847bb4d4f11c8cfc6b38fcb05b10023cdc08) 

This commit will update the revisions of `Bulkrax` and `AllinsonFlex` to
the latest versions to bring in the views changes for the admin
dashboard sidebar.  Also, this commit moves the js require for Bulkrax
up above the jQuery requires to fix the issue with the admin jQuery
throwing errors.  This was likely due to a recent change to make Bulkrax
work better with Hydra.

Ref:
  - https://github.com/samvera-labs/bulkrax/pull/836
  - https://github.com/samvera-labs/allinson_flex/pull/120
  - https://github.com/samvera-labs/bulkrax/pull/838

---

[🧹 Update translations](https://github.com/scientist-softserv/utk-hyku/pull/486/commits/a335c40f53309973a3675a965eb412c10dc443be) 

This commit will run Google translations from en to all the other
languages supported by Hyrax.

# Expected Behavior Before Changes

Shrinking the admin dashboard's width less than 768px would not affect the sidebar.  Hovering over the sidebar links did not pop up any text so this was challenging when the sidebar was minimized.

# Expected Behavior After Changes

Sidebar auto minimizes after when the viewing width is less than 768px.  Hovering over the sidebar links now will give you the name of the link so it'll be easier to tell what you're clicking on when the sidebar is minimized.

# Screenshots / Video

https://github.com/scientist-softserv/utk-hyku/assets/19597776/023bbb07-194d-4cee-bd74-022ffe1c25e3

### CODE REVIEW

File: app/views/hyrax/dashboard/sidebar/_activity.html.erb [hunk 1]
```diff
@@ -4,12 +4,15 @@
     <%= menu.collapsable_section t('hyrax.admin.sidebar.repository_activity'),
                                  icon_class: "fa fa-line-chart",
                                  id: 'collapseRepositoryActivity',
-                                 open: menu.repository_activity_section? do %>
-      <%= menu.nav_link(hyrax.dashboard_path) do %>
+                                 open: menu.repository_activity_section?,
+                                 title: t('hyrax.admin.sidebar.repository_activity') do %>
```
Comment (ShanaLMoore) [inline app/views/hyrax/dashboard/sidebar/_activity.html.erb:8]: If we aren't adding alt tags now, could you make sure there's a separate ticket for it? I'm pretty sure, given the other alt tag failure, that not having alt tags on the menu will also be an issue.

File: config/locales/en.yml [hunk 2]
```diff
@@ -152,10 +152,12 @@ en:
             fonts: "Fonts"
             themes: "Themes"
       sidebar:
+        account: Account
```
Comment (ShanaLMoore) [inline config/locales/en.yml:155]: Should we go ahead and run the translations?

Comment (kirkkwang) [reply config/locales/en.yml:155]: That sounds good! I'm not sure how to do that but I'll ask around.

File: app/views/hyrax/dashboard/sidebar/_activity.html.erb (hunk 1 above)
Comment (kirkkwang) [reply app/views/hyrax/dashboard/sidebar/_activity.html.erb:8]: In a Bulkrax PR Jeremy [reminded me ](https://github.com/samvera-labs/bulkrax/pull/837#discussion_r1269387732) that alt tags are technically for `` tags

File: config/locales/en.yml (hunk 2 above)
Comment (ShanaLMoore) [reply config/locales/en.yml:155]: there's a playbook article, i'll find and link it

Comment (ShanaLMoore) [reply config/locales/en.yml:155]: https://playbook-staging.notch8.com/en/samvera/hyrax/how_to/generate_translations

File: app/views/hyrax/dashboard/sidebar/_activity.html.erb (hunk 1 above)
Comment (ShanaLMoore) [reply app/views/hyrax/dashboard/sidebar/_activity.html.erb:8]: Ahh that's right. Ok!

File: config/locales/en.yml (hunk 2 above)
Comment (ShanaLMoore) [reply config/locales/en.yml:155]: @kirkkwang It looks like the value of GOOGLE_TRANSLATE_API_KEY is in 1password. Hopefully it's up to date.

Comment (kirkkwang) [reply config/locales/en.yml:155]: Great thank you, i'll give that a shot, i remember something about it being out of money but i'm hoping that's been resolved

Comment (kirkkwang) [reply config/locales/en.yml:155]: i've made another commit for running the translations

**Review by ShanaLMoore — APPROVED** [review]

**Pull request merged**